src/node_i18n.h

```diff
@@ -42,7 +42,7 @@ bool InitializeICUDirectory(const std::string& path);
 
 void SetDefaultTimeZone(const char* tzid);
 
-enum idna_mode {
+enum class idna_mode {
```

Comment:
I suggest removing the `IDNA_` prefix from the members of `idna_mode` since the scope is changing. For example, `idna_mode::IDNA_STRICT` should just be `idna_mode::STRICT`.

Reply:
I think good suggestion from a clean code point of view. Removed `IDNA_` prefix. (bc0541f)